### PR TITLE
DOC: logging

### PR DESCRIPTION
**Pull request by UranusSeven**
## What do these changes do?
Add logging doc.

## Related issue number

Fixes #xxxx

## Check code requirements

- [ ] tests added / passed (if needed)
- [ ] Ensure all linting tests pass

### CODE REVIEW

File: doc/source/user_guide/logging.rst [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+=======
+Logging
+=======
+
+This document will explain Xorbits’s logging system. By default, Xorbits logs are written to stderr
+as well as files.
+
+Log to stderr
+-------------
+The default log level for stderr is set to ``WARNING`` to to limit the amount of log output.
```
Comment (aresnow1) [inline doc/source/user_guide/logging.rst:12]:
```suggestion
The default log level for stderr is set to ``WARNING`` to limit the amount of log output.
```

**Comment by UranusSeven** [comment]
> Do we need to explain the behavior in the IPython environment?

Actually, there's no difference except IPython env uses the root logger's level and format for the stream handler. And there's an example for this scenario.

**Comment by aresnow1** [comment]
Doc build failed, is it related to the changes?

**Comment by UranusSeven** [comment]
> Doc build failed, is it related to the changes?

Probably, let me have a look.

**Review by ChengjieLi28 — APPROVED** [review]

**Review by aresnow1 — APPROVED** [review]
LGTM

**Pull request merged**